presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -26,21 +26,24 @@
 
 set dshot_bidir = OFF
 set simplified_gyro_filter = OFF
```

Comment:
Don't need to disable as we now are able to use different filter combinations. Prefer a tuning with sliders. It this possible in your tune?

Reply:
@haslinghuis, i think we should not mandate sliders tuning, but you are right - highly encouraged :)

Reply:
> @SupaflyFPV so you intentionally don't want to use filter sliders?

yes of course - it's intentional.  There is no need for the user to use sliders-I set them where they should be thats it job done. Only change is to switch off Gyro Low Pass 2 which is mentioned in description.

Reply:
Ok I can't do what I want whilst also using the filter sliders.

I want two static DTerm LPF PT1 - both are at conservative settings so there is no issue there.

I want gyro LPF predominantly from a single PT1 dynamic. The second Gyro LPF2 is mainly used as an anti alias filter at 1000k (and switched off if user has F7/H7 and runs 8k).

It's a simple, robust and logical setup IMO but it does not fit with the current slider config.

Reply:
I should note these settings on filters have already been accepted in the current version of the preset